packages/untaped-core/tests/unit/test_settings.py

```
@@ -161,3 +162,13 @@ def test_awx_api_prefix_must_start_and_end_with_slash() -> None:
         AwxSettings(api_prefix="api/v2/")
     with pytest.raises(ValidationError):
         AwxSettings(api_prefix="/api/v2")
+
+
+def test_resolve_config_path_honours_env_var(monkeypatch: pytest.MonkeyPatch) -> None:
+    monkeypatch.setenv("UNTAPED_CONFIG", "/tmp/custom-config.yml")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/custom-config.yml"

(S108)